src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -373,6 +421,7 @@ static bool equalMetadataValues(BSONObj& md1, BSONObj& md2)
 */
 static bool equalMetadataVectors(BSONObj& mdV1, BSONObj& mdV2)
```

Comment:
Probably that issue already exist, I think this is not the first time we talk about this...